### ISSUE TITLE
Ads

### ISSUE THREAD

**Issue by PSI-Rockin** [wheremyfoodat/Panda3DS#479]
Hello I think it would be a great idea to add ads to the emulator it would be great because you could make a lot of money which you could use to fund development so that the emulator runs at full speed on my Pentium II thank you and have a nice day

**Comment by wheremyfoodat** [comment]
I agree. I think we should spice it up using naughty xxx ads since those make more revenue. Thoughts?

**Comment by wheremyfoodat** [comment]
GET TITAN GEL TO ENLARGE YOUR SCHLONG

**Comment by OFFTKP** [comment]
I like this Idea. I came up with an intuitive code name for this project: "Ads on Panda3DS"

I painted a quick image to summarize our plan:
![image](https://github.com/wheremyfoodat/Panda3DS/assets/21157395/4810c91f-c64f-4ec2-92b8-e5101279a143)

**Comment by PSI-Rockin** [comment]
@OFFTKP Wow!!!!! Your idea is great!!!! You should get this submitted to the developers so that I can enjoy ads at full speed

**Comment by eepykami** [comment]
Panda3DS After Dark version when

**Comment by wheremyfoodat** [comment]
OK so I am adding ads but I do not have a bank account. Can somebody give me their credit card info (16 digits in the front, expiration date and 3-digit code in the back) for use with our ad network?

**Comment by PSI-Rockin** [comment]
Sure it's ****-****-****-**** **/** ***

**Comment by wheremyfoodat** [comment]
I thought that I'd been hurt before
But no one's ever left me quite this sore
Your words cut deeper than a knife
Now I need someone to breathe me back to life
Got a feeling that I'm going under
But I know that I'll make it out alive
If I quit calling you my lover
Move on
You watch me bleed until I can't breathe
Shaking, falling onto my knees
And now that I'm without your kisses
I'll be needing stitches
Tripping over myself
Aching, begging you to come help
And now that I'm without your kisses
I'll be needing stitches
Just like a moth drawn to a flame
Oh, you lured me in, I couldn't sense the pain
Your bitter heart, cold to the touch
Now I'm gonna reap what I sow
I'm left seeing red on my own
Got a feeling that I'm going under
But I know that I'll make it out alive
If I quit calling you my lover
Move on
You watch me bleed until I can't breathe
Shaking, falling onto my knees
And now that I'm without your kisses
I'll be needing stitches
Tripping over myself
Aching, begging you to come help
And now that I'm without your kisses
I'll be needing stitches
Needle and the thread, gotta get you outta my head
Needle and the thread, gonna wind up dead
Needle and the thread, gotta get you outta my head
Needle and the thread, gonna wind up dead (gonna wind up dead)
Needle and the thread, gotta get you outta my head (gotta get you outta my head)
Needle and the thread, gonna wind up dead (wind up dead)
Needle and the thread, gotta get you outta my head, get you outta my head
You watch me bleed until I can't breathe
Shaking, falling onto my knees (falling on my knees)
And now that I'm without your kisses (without you)
I'll be needing stitches (and I'll be needing stitches)
Tripping over myself
Aching, begging you to come help (begging baby, please)
And now that I'm without your kisses
I'll be needing stitches
And now that I'm without your kisses
I'll be needing stitches
And now that I'm without your kisses
I'll be needing stitches

**Comment by PSI-Rockin** [comment]
omg how do i use github

**Comment by eepykami** [comment]
* * * * * * * * * * * * * * * * * / * * / * * / * * * 
GitHub seems to be automatically censoring my credit card information. Can someone else confirm this?

**Comment by wheremyfoodat** [comment]
@PSI-Rockin Please follow these steps

![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/cbd9b2a1-74b6-4eed-a16f-6a1a334c6279)
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/86964896-bbd7-4e3c-8392-6e782453e652)
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/a12f8fe8-3873-498b-b398-31bc385aa9e4)
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/10692d2f-7be8-40c1-a99c-79d0d189ee07)

**Comment by OFFTKP** [comment]
HlektronikiPonos

**Comment by PSI-Rockin** [comment]
@wheremyfoodat No matter how many times I try it won't let me delete your account. I think you have to do this for me

**Comment by wheremyfoodat** [comment]
Come over

**Comment by JaffaCat** [comment]
please add subway surfers ads to the emulator, it will increase attention spans of users using it

**Comment by Auxy6858** [comment]
Someone make a tiktok app for the 3DS

**Comment by DaniElectra** [comment]
It's been one year and we still don't have ads, what kind of emulator is this?

**Comment by Auxy6858** [comment]
Terrible, I'm forking both this project and the unity ads repo and combining the two to create THE most invasive ads known to man.
